packages/TermsAndConditions/FootnoteLink/FootnoteLink.md

```diff
@@ -0,0 +1,67 @@
+### Usage criteria
+
+- If only one annotation is required, when possible place elaborative copy in-page with the language or element in question
+- For `PriceLockup` all the numbers should appear together below the `bottomText`
+- Must be wrapped by a `Text`, `Heading`, or `DisplayHeading` component
+  - When using `FootnoteLink` in a `PriceLockup`, `PriceLockup` will automatically wrap all `bottomText` in a `Text` component
+
+```jsx
+<Card>
+  <Heading level="h4">For Non TELUS mobility customers</Heading>
+
+  <Paragraph>$30 per month</Paragraph>
+
+  <Paragraph>
+    <Text>
```

Comment:
The reason why it is wrapped with `Text` is because we need to take advantage of `Text` `<sup>` styles. Until we can style `<sup>` in `Paragraph` we will need to wrap with `Text`

Reply:
Note to selves: let's revisit this after the rest of these comments are addressed.

Reply:
We will resolve this after TDS V3 release.